README.md

```diff
@@ -86,6 +86,30 @@ Hello, world!
 [Cranelift]: https://github.com/bytecodealliance/wasmtime/blob/master/cranelift/README.md
 [embedded]: https://bytecodealliance.github.io/wasmtime/embed.html
 
+## Language Support
+
+You can use Wasmtime from a variety of different languages through embeddings of
+the implementation:
+
+* **[Rust]** - the [`wasmtime` crate]
+* **[C]** - the [`wasm.h`], [`wasi.h`], and [`wasmtime.h`] headers
+* **[Python]** - the [`wasmtime` PyPI package]
+* **[.NET]** - the [`Wasmtime` NuGet package]
+* **[Go]** - the [wasmtime-go repository]
+
+[Rust]: https://bytecodealliance.github.io/wasmtime/embed-rust.html
```

Comment:
Would it make sense to rename `embed-rust.html` to `lang-rust.html` and `embed.md` to `lang.md`, and remove `embed-c.md`, so that we don't have some languages using `embed-*` and others using `lang-*`?